docs/tools/finetune_gen.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""
+Update test_spec.rst with test metadata
+"""
+import ast
+import json
+import os
+from pathlib import Path
+from typing import Dict
+
+from .doc_generator import TESTS, ClassVisitor, FuncVisitor, extract_metadata, load_path
+
+base_path = Path(__file__).parent
+file_path = (base_path / "../_build/html/finetune_cases.jsonl").resolve()
+system_prompt = "You are a helpful assistant to write Python code for linux validation."
+user_prompt = (
+    "Please bases on the test case description to write a LISA test case.\n\n\n"
```

Comment:
`Please write a LISA test case based on the test case description.`

Reply:
fixed